packages/whale-api-client/src/api/consortium.ts

```diff
@@ -15,10 +15,52 @@ export class Consortium {
   async getAssetBreakdown (): Promise<AssetBreakdownInfo[]> {
     return await this.client.requestData('GET', 'consortium/assetbreakdown')
   }
+
+  /**
+   *  Gets the transaction history of consortium members.
+   *
+   * @param {number} start The starting index for pagination
+   * @param {number} [limit] How many transactions to fetch
+   * @param {string} [search] Search term, can be a transaction id, member/owner address or member name
+   * @return {Promise<ConsortiumTransactionResponse[]>}
+    */
+  async getTransactionHistory (pageIndex: number, limit: number, search?: string): Promise<ConsortiumTransactionResponse> {
```

Comment:
```suggestion
  async getTransactionHistory (pageIndex?: number, limit?: number, search?: string): Promise<ConsortiumTransactionResponse> {
```
Can we set those parameters to optional? since the API also no expect those to be mandatory.

Reply:
Thanks, included here: [`11c1e50`](https://github.com/JellyfishSDK/jellyfish/pull/1868/commits/11c1e502c77f6aab8d11a02f7572bd3aa55ac72b)